<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtserterm</class>
 <widget class="QMainWindow" name="qtserterm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Qt Serial Terminal</string>
  </property>
  <property name="windowIcon">
   <iconset theme="terminal"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpSerialPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
    </property>
    <property name="title">
     <string>Serial Port</string>
    </property>
    <widget class="QLabel" name="lblBaud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblParams">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8 Bit - No Parity - Stop 1 - No Flow</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPortScan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPortOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>461</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
    </property>
    <property name="title">
     <string>Terminal</string>
    </property>
    <widget class="QPlainTextEdit" name="txtSerTerm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpSerialData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
    </property>
    <property name="title">
     <string>Serial Data</string>
    </property>
    <widget class="QLabel" name="lblLocalEcho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Local Echo</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLocalEcho">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Disable</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enable</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbShowEnd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Disable</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enable</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblShowEnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Ending</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpFileDump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
    </property>
    <property name="title">
     <string>Save Data</string>
    </property>
    <widget class="QLineEdit" name="txtFilePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFileSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFilePath">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>31</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder-open"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AutoSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
    </property>
    <property name="title">
     <string>String Send</string>
    </property>
    <widget class="QLineEdit" name="txtDataSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblLineEnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Line Ending</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLineEnd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CR/LF</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnAutoStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Audo Send Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStringSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtInterval">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAutoInterval">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval (ms)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSendWait">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wait (ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSendWait">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPortInfo"/>
    <addaction name="actionPlotter"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="sttbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="qtassistant"/>
   </property>
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionPlotter">
   <property name="icon">
    <iconset theme="window-new"/>
   </property>
   <property name="text">
    <string>Open Plotter</string>
   </property>
  </action>
  <action name="actionPortInfo">
   <property name="icon">
    <iconset theme="system-devices-panel-informations"/>
   </property>
   <property name="text">
    <string>Port Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
